sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -510,5 +510,13 @@ export class ChatClient {
     this.signalingClient.on("participantsRemoved", (payload) => {
       this.emitter.emit("participantsRemoved", payload);
     });
+
+    // this.signalingClient.on("streamingChatMessageStarted", (payload) => {
```

Comment:
Why are these commented?

Reply:
ah, the PR waits on signaling and is work in progress

Reply:
Yes - this PR will not be merged; it's just to highlight the changes coming from the underlying signaling.